apps/src/templates/projects/PersonalProjectsPublishedCell.jsx

```diff
@@ -16,13 +17,32 @@ class PersonalProjectsPublishedCell extends Component {
     publishMethod: PropTypes.oneOf([publishMethods.CHEVRON, publishMethods.BUTTON]).isRequired,
     showPublishDialog: PropTypes.func.isRequired,
     unpublishProject: PropTypes.func.isRequired,
+    userId: PropTypes.string,
   };
 
   onPublish = () => {
+    firehoseClient.putRecord(
+      {
+        study: 'project-publish',
+        study_group: 'publish-button',
+        event: 'publish',
+        user_id: this.props.userId,
+        channel_id: this.props.projectId,
+      }
+    );
     this.props.showPublishDialog(this.props.projectId, this.props.projectType);
   };
 
   onUnpublish = () => {
+    firehoseClient.putRecord(
+      {
+        study: 'project-publish',
+        study_group: 'publish-button',
+        event: 'unpublish',
+        user_id: this.props.userId,
+        channel_id: this.props.projectId,
```

Comment:
For new fields other than `user_id`, I've just used the `data_json` field, so something like:
```
data_json: JSON.stringify({ channel_id: this.props.projectId }),
```